docs/querying/basics.md

```diff
@@ -87,16 +88,53 @@ Examples:
     0x8f
     -Inf
     NaN
-	
 
-As of version 2.54, float literals can also be represented using the syntax of time durations, where the time duration is converted into a float value corresponding to the number of seconds the time duration represents. This is an experimental feature and might still change.
+Additionally, underscores (`_`) can be used in between decimal or hexadecimal
+digits to improve readability.
 
 Examples:
 
-    1s # Equivalent to 1.0
-    2m # Equivalent to 120.0
-    1ms # Equivalent to 0.001
- 
+    1_000_000
+	.123_456_789
+	0x_53_AB_F3_82
+
+Float literals are also used to specify durations in seconds. For convenience,
```

Comment:
How about adding a "sometimes" here, somehow it feels like it could make things less confusing:

```suggestion
Float literals are also sometimes used to specify durations in seconds. For convenience,
```

Reply:
Yeah, I guess you're right, or at least I don't feel strongly about this, also fine to keep it as it is.